AUTHORS.md

```diff
@@ -325,6 +326,7 @@ Authors are sorted alphabetically.
 * Shannon Green
 * Shaun Thompson
 * Shaun Walbridge
+* Shaun Walbridge
```

Comment:
FYI this is a different Shaun Walbridge so the duplicate is expected

Should we change the `AUTHORS.md` formatting to use `{name} ({github})` to help distinguish?

Reply:
Yeah, I think that's a good idea @kenodegard!

Reply:
(let's do this after the release)

Reply:
Agreed